hack/csi_client/README.md

```diff
@@ -16,9 +16,7 @@ As of now, only two types of requests are supported:
 
 If the SMB share requires a username and password, these can be provided by using the flags `smbUsername` and `smbPassword`:
 ```Powershell
-.\csi-client.exe --endpoint="C:\tmp\csi.sock" --request="nodepublish" --shareAddr="localhost"
---shareName="SMBShare" --targetPath="C:\SMBShareMount"
---smbUsername="smbuser" --smbPassword="foobar"
+.\csi-client.exe --endpoint="C:\tmp\csi.sock" --request="nodepublish" -- volumeAttr="ip=localhost,volume=SMBShare" --targetPath="C:\SMBShareMount" --secrets="smbUser=smbuser,smbPassword=foobar"
```

Comment:
Can you add a note in the README about the limitation that comma is used as a delimeter and therefore cannot be a character in the secret?